devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -443,6 +448,19 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
         return false;
     }
 
+    if (!(config.check("jointList") && config.find("jointList").isList())) {
```

Comment:
May be adding in the main comment a sentence as below will be easy to track the changes.

> Add optional `jointList` parameter for `HumanStateProvider` device

Reply:
R